lib/monitoring/aws-cloudwatch/SyntheticsCanaryMonitoring.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  GraphWidget,
+  HorizontalAnnotation,
+  IWidget,
+} from "monocdk/aws-cloudwatch";
+import {
+  BaseMonitoringProps,
+  CountAxisFromZero,
+  DefaultGraphWidgetHeight,
+  DefaultSummaryWidgetHeight,
+  ErrorAlarmFactory,
+  ErrorCountThreshold,
+  ErrorRateThreshold,
+  ErrorType,
+  HalfWidth,
+  LatencyAlarmFactory,
+  LatencyThreshold,
+  LatencyType,
+  MetricWithAlarmSupport,
+  Monitoring,
+  MonitoringScope,
+  RateAxisFromZero,
+  ThirdWidth,
+  TimeAxisMillisFromZero,
+} from "../../common";
+import {
+  MonitoringHeaderWidget,
+  MonitoringNamingStrategy,
+} from "../../dashboard";
+import {
+  SyntheticsCanaryMetricFactory,
+  SyntheticsCanaryMetricFactoryProps,
+} from "./SyntheticsCanaryMetricFactory";
+
+export interface SyntheticsCanaryMonitoringProps
+  extends SyntheticsCanaryMetricFactoryProps,
+    BaseMonitoringProps {
+  readonly addAverageLatencyAlarm?: Record<string, LatencyThreshold>;
+  readonly add4xxErrorCountAlarm?: Record<string, ErrorCountThreshold>;
+  readonly add4xxErrorRateAlarm?: Record<string, ErrorRateThreshold>;
+  readonly add5xxFaultCountAlarm?: Record<string, ErrorCountThreshold>;
+  readonly add5xxFaultRateAlarm?: Record<string, ErrorRateThreshold>;
+}
```

Comment:
Change this to extract `SyntheticsCanaryMonitoringOptions` and `SyntheticsCanaryMonitoringProps` so as to use then in aspect monitoring.

Reply:
Will do!